README.md

```diff
@@ -1,20 +1,38 @@
 # admiralonco <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
 Oncology extension package for ADaM in R Asset Library `{admiral}`
 
-### Purpose
+## Purpose
 
 To provide a complementary (to `{admiral}`) toolbox that enables to develop oncology disease area specifics.
 
-### Scope
+## Scope
 
 * Build a toolbox of re-usable functions and utilities to create oncology-specific ADaM datasets in R in a modular manner.
 * All functions are created based upon the ADaM Implementation Guide and aim to facilitate the programming of ADaM dataset standards.
 * Initially the package will focus on the most common efficacy endpoint needs, but over time we will look to add extra areas such as:
 endpoints for targeted indications, baseline disease characteristics, common sensitivity analyses and questionnaires.
 
-### R Versions
+## Expectations
+
+* `{admiralonco}` is expected to complement `{admiral}` and provide functions to help with the creation of the efficacy endpoints required for oncoly ADaMs.
+
+* Please also refer to the [`{admiral}` expectations](https://pharmaverse.github.io/admiral/index.html#expectations)
```

Comment:
this can be removed as we're re-working admiral README now and no longer have an expectations section. see the PR i added today and mentioned in the dev slack channel